pkg/controllers/jobset_controller.go

```diff
@@ -255,10 +255,15 @@ func (r *JobSetReconciler) resumeJobSetIfNecessary(ctx context.Context, js *jobs
 				log.Error(nil, "job missing ReplicatedJobName label")
 			}
 			job.Spec.Suspend = pointer.Bool(false)
-			job.Status.StartTime = nil
 			if err := r.Update(ctx, job); err != nil {
 				return err
 			}
+			if job.Status.StartTime != nil {
+			  job.Status.StartTime = nil
+			  if err := r.Status().Update(ctx, job); err != nil {
+			 	return err
+			  }
+			}
```

Comment:
@mcariatm it seems that there is a missing bracket

Reply:
no, it was due to the indentation being too short.

Reply:
should be ok now....